docs/file_formats.md

```diff
@@ -0,0 +1,64 @@
+
+### Numpy arrays (\*.npy, \*.npz)
+Caiman can load files in numpy's native array file format. This will likely only have been produced by some other software (that you found or wrote) that preprocessed your original data and saved it, and that software was likely written in Python because few things not written in Python might've written this Python-specific format. It's fast and easy to work with this format
+
+### HDF5/N5/zarr (\*.h5, \*.hdf5, \*.n5, \*.zarr)
+Caiman can easily work with files using these container formats for grid data. In many ways they're ideal - the format is regular, parsing them is easy, a variety of programming languages can work with them, and reads/writes are very fast.  HDF5 files are really files; n5 and zarr files are directories with a hierarchy beneath, which can make moving them between hosts annoying (but their performance is even better). Because all these formats can have multiple datasets inside a single store, if there is more than one dataset present you'll need to tell Caiman what dataset you want to work with when you first start processing the data (using the var\_name\_hdf5 flag to cm.load() and related functions). If there is only one dataset present in the file, Caiman will default to using that one and doesn't need to be told.
+
+### Memmapped Numpy Array (\*.mmap)
+Caiman internally uses this data format, and you can use it too with Caiman, although it is really unlikely you'll come across this format elsewhere and we don't recommend you generate it outside of Caiman (we also may eventually change this format or remove it)
+
+### Scanbox Format (\*.sbx)
+This format is a common format used by the Scanbox application (written in Matlab).  It is a variant of the .mat file (above) with a particular internal structure and interpretation of values.  If you need to use it, Caiman may be able to load it, but we don't recommend you use this format otherwise. The scipy.io.loadmat() functions (with a lot of supporting code) are used to support this.
+
+### SIMA (\*.sima)
+Support for this format was removed in an earlier version of Caiman.
+
+### Zipfile full of images (\*.zip)
+The default functions don't support this. Read the docs for movies.py:from\_zip\_file\_to\_movie() to use this support
+
+## Functions
+load() supports a broader range of file formats than load\_iter().
+(TODO fill in this section)
+
+On bit-depth for the formats
+(TODO fill in this section)
+
+## Conversion tools
+ImageJ
+(tiff file converters)
+
```

Comment:
What about adding an example of how to convert something that can be loaded as a numpy array into a caiman-compatible format? Might be useful for beginners.